coalib/bears/Bear.py

```diff
@@ -136,6 +146,23 @@ def can_detect(cls):
         """
         return cls.CAN_DETECT | cls.CAN_FIX
 
+    @classproperty
+    def maintainers(cls):
+        """
+        :return: A set containing MAINTAINERS if specified, else takes
+                 AUTHORS by default.
+        """
+        return cls.AUTHORS if cls.MAINTAINERS == set() else cls.MAINTAINERS
```

Comment:
I'm wondering if maintainers should be authors + maintainers (just in case they are different)

Not sure.